### PR TITLE
Encounter cards for BBT

### PR DESCRIPTION
**Pull request by zzorba**

### CODE REVIEW

File: pack/tcu/bbt_encounter.json [hunk 1]
```diff
@@ -0,0 +1,404 @@
+[
+	{
+		"back_text": "Hard / Expert\n[skull]: -X. X is the amount of doom on Azathoth, to a minimum of 2.\n[cultist]: Reveal another token. If you fail, search the encounter deck and discard pile for a [[Cultist]] enemy and draw it. Shuffle the encounter deck.\n[tablet]: -3. If you fail, Azathoth attacks you.\n[elder_thing]: -6. If your modified skill value for this test is 0, place 1 doom on Azathoth.",
+		"code": "05025",
+		"double_sided": true,
+		"encounter_code": "before_the_black_throne",
+		"faction_code": "mythos",
```
Comment (drawntotheflame) [inline pack/tcu/bbt_encounter.json]: Please add "encounter_position" to all cards.

File: pack/tcu/bbt_encounter.json [hunk 2]
```diff
@@ -0,0 +1,404 @@
...
+		"health": null,
+		"illustrator": "Brian Valenzuela",
+		"is_unique": true,
+		"name": "Azathoth",
+		"pack_code": "bbt",
+		"position": 346,
+		"quantity": 1,
+		"subname": "The Primal Chaos",
+		"text": "Azathoth is oblivious and omnipotent. It is immune to all player card effects and investigator actions, and cannot be defeated by any means.\n<b>Forced</b> - If there is 10 or more doom on Azathoth: The Blind Idiot God rises from its throne in the ultimate void of Chaos and devours the universe. <b>(→R1)</b>",
```
Comment (drawntotheflame) [inline pack/tcu/bbt_encounter.json]: Please change 'Chaos' to 'chaos'.

File: pack/tcu/bbt_encounter.json [hunk 3]
```diff
@@ -0,0 +1,404 @@
...
+		"quantity": 3,
+		"text": "<b>Revelation</b> - Test [willpower] (4). If you fail, attach Ultimate Chaos to Azathoth. If you fail by 2 or more, take 1 damage and 1 horror. If you fail by 3 or more, Ultimate Chaos gains surge. Cannot be canceled.\nIf there are 3 copies of Ultimate Chaos attached to Azathoth, discard them and either place 1 doom on Azathoth, or Azathoth attacks each investigator in player order. Cannot be canceled.",
+		"traits": "Power.",
+		"type_code": "treachery"
+	},
+	{
+		"code": "05343",
+		"encounter_code": "before_the_black_throne",
+		"faction_code": "mythos",
+		"flavor": "There was the immemorial figure of the deputy or messenger of hidden and terrible powers...\n– H. P. Lovecraft, The Dreams in the Witch House",
```
Comment (drawntotheflame) [inline pack/tcu/bbt_encounter.json]: Do we like to do underline tags for story names?

File: pack/tcu/bbt_encounter.json [hunk 4]
```diff
@@ -0,0 +1,404 @@
...
+		"quantity": 1,
+		"shroud": 2,
+		"text": "<b>Forced</b> - At the end of the round: Add clues to Cosmic Ingress until it has 3 clues on it.\nEach [[Void]] location gains: \"[action]: Shuffle this location into the Cosmos, moving each investigator and enemy that was at this location to Cosmic Ingress.\"",
+		"traits": "Otherworld.",
+		"type_code": "location"
+	},
+	{
+		"back_illustrator": "Uwe Jarling",
+		"back_name": "Comsos",
+		"back_flavor": "The stars danced mockingly, almost shifting now and then to form pale signs of doom that one might wonder one had not seen and feared before; and ever the winds of aether howled of vague blackness and loneliness beyond the cosmos.\n– H. P. Lovecraft, The Dream-Quest of Unknown Kadath",
```
Comment (drawntotheflame) [inline pack/tcu/bbt_encounter.json]: Do we like to do underline tags for story names? Happy to leave it if you don't fancy adding it across all the cosmos locations!

File: pack/tcu/bbt_encounter.json [hunk 5]
```diff
@@ -0,0 +1,404 @@
...
+		"double_sided": true,
+		"encounter_code": "before_the_black_throne",
+		"faction_code": "mythos",
+		"illustrator": "Stephen Somers",
+		"name": "Pathway into Void",
+		"pack_code": "bbt",
+		"position": 340,
+		"quantity": 2,
+		"shroud": 4,
+		"text": "<b>Cosmos</b> - Connect in a direction of your choice.\n<b>Forced</b> - After you enter Pathway into Void: You must either discard 1 card from your hand or take 1 damage.\n[action] Remove 1 resource from the scenario reference card: Move Pathway into Void to any adjacent spot, placing an empty space in the spot it previously occupied. (Group limit once per turn.)",
```
Comment (drawntotheflame) [inline pack/tcu/bbt_encounter.json]: Please change the group limit from 'once per turn' to 'once per round'.

File: pack/tcu/bbt_encounter.json [hunk 6]
```diff
@@ -0,0 +1,404 @@
...
+		"shroud": 4,
+		"text": "<b>Forced</b> - At the end of the round: Add clues to Court of the Great Old Ones until it has 6 clues on it.\nEach [[Void]] location gains: \"[action]: Shuffle this location into the Cosmos, moving each investigator and enemy that was at this location to Court of the Great Old Ones.\"",
+		"traits": "Otherworld.",
+		"type_code": "location"
+	},
+	{
+		"back_illustrator": "Uwe Jarling",
+		"back_name": "Comsos",
+		"back_flavor": "The stars danced mockingly, almost shifting now and then to form pale signs of doom that one might wonder one had not seen and feared before; and ever the winds of aether howled of vague blackness and loneliness beyond the cosmos.\n– H. P. Lovecraft, The Dream-Quest of Unknown Kadath",
+		"back_traits": "Otherworld.",
```
Comment (drawntotheflame) [inline pack/tcu/bbt_encounter.json]: You can delete this line - back and front match.

File: pack/tcu/bbt_encounter.json [hunk 7]
```diff
@@ -0,0 +1,404 @@
...
+		"clues": 2,
+		"code": "05335",
+		"double_sided": true,
+		"encounter_code": "before_the_black_throne",
+		"flavor": "...the ultimate nighted throne of the daemon Azathoth in the formless central void.\n– H. P. Lovecraft, The Dream-Quest of Unknown Kadath",
```
Comment (drawntotheflame) [inline pack/tcu/bbt_encounter.json]: Possible underline story name...

File: pack/tcu/bbt_encounter.json [hunk 8]
```diff
@@ -0,0 +1,404 @@
...
+		"shroud": 3,
+		"text": "<b>Forced</b> - At the end of the round: Add clues to Hideous Palace until it has 4 clues on it.\nEach [[Void]] location gains: \"[action]: Shuffle this location into the Cosmos, moving each investigator and enemy that was at this location to Hideous Palace.\"",
+		"traits": "Otherworld.",
+		"type_code": "location"
+	},
+	{
+		"back_illustrator": "Uwe Jarling",
+		"back_name": "Comsos",
+		"back_flavor": "The stars danced mockingly, almost shifting now and then to form pale signs of doom that one might wonder one had not seen and feared before; and ever the winds of aether howled of vague blackness and loneliness beyond the cosmos.\n– H. P. Lovecraft, The Dream-Quest of Unknown Kadath",
+		"back_traits": "Otherworld.",
```
Comment (drawntotheflame) [inline pack/tcu/bbt_encounter.json]: You can delete this line.

File: pack/tcu/bbt_encounter.json [hunk 9]
```diff
@@ -0,0 +1,404 @@
...
+		"shroud": 2,
+		"text": "<b>Forced</b> - At the end of the round: Add clues to Cosmic Ingress until it has 3 clues on it.\nEach [[Void]] location gains: \"[action]: Shuffle this location into the Cosmos, moving each investigator and enemy that was at this location to Cosmic Ingress.\"",
+		"traits": "Otherworld.",
+		"type_code": "location"
+	},
+	{
+		"back_illustrator": "Uwe Jarling",
+		"back_name": "Comsos",
+		"back_flavor": "The stars danced mockingly, almost shifting now and then to form pale signs of doom that one might wonder one had not seen and feared before; and ever the winds of aether howled of vague blackness and loneliness beyond the cosmos.\n– H. P. Lovecraft, The Dream-Quest of Unknown Kadath",
+		"back_traits": "Otherworld.",
```
Comment (drawntotheflame) [inline pack/tcu/bbt_encounter.json]: You can delete this line - traits match front and back.

File: pack/tcu/bbt_encounter.json [hunk 10]
```diff
@@ -0,0 +1,404 @@
...
+		"name": "What Must Be Done",
+		"pack_code": "bbt",
+		"position": 331,
+		"quantity": 1,
+		"stage": 3,
+		"text": "[action] Spend X clues: Draw the top X cards of the Cosmos. Choose one to put into play using its \"<b>Cosmos</b> -\" instructions, and move to it. Shuffle the rest back into the Cosmos.\n<b>Objective</b> - If the lead investigator is at The Black Throne and there are no clues on it, advance.",
+		"type_code": "act"
+	},
+	{
+		"back_flavor": "...that eldritch scaly monster bore its helpless rider; hurtling and shooting, cleaving the uttermost rim and spanning the outermost abysses; leaving behind the stars and the realms of matter, and darting meteor-like through stark formlessness toward those inconceivable, unlighted chambers beyond Time...\n– H. P. Lovecraft, The Dream-Quest of Unknown Kadath",
```
Comment (drawntotheflame) [inline pack/tcu/bbt_encounter.json]: Possible underline story line...

File: pack/tcu/bbt_encounter.json [hunk 11]
```diff
@@ -0,0 +1,404 @@
...
+		"back_name": "The End",
+		"back_text": "Check Campaign Log.\n- If at least two of the following are true, you may proceed to <b>(→R2)</b>: <i>You accepted your fate</i>, <i>Mesmirizing Flute</i> is listed under \"Mementos Discovered,\" and <i>Ritual Components</i> is listed under \"Mementos Discovered.\"\n- If at least two of the following are true, you may proceed to <b>(→R3)</b>: <i>You rejected your fate</i>, <i>Scrap of Torn Shadow</i> is listed under \"Mementos Discovered,\" and <i>Wisp of Spectral Mist</i> is listed under \"Mementos Discovered.\"\n- Otherwise, <b>(→R4)</b>.",
+		"clues": 0,
+		"code": "05331",
+		"double_sided": true,
+		"encounter_code": "before_the_black_throne",
+		"faction_code": "mythos",
+		"flavor": "You are close now. You can feel the churning and twisting of aeons in the emptiness around you. Here, in the nucleus of all things—the very center of chaos itself—your objective, and your ultimate fate, lie at hand.",
```
Comment (drawntotheflame) [inline pack/tcu/bbt_encounter.json]: Please can you add spaces either side of the dashes? (There are two.)

File: pack/tcu/bbt_encounter.json [hunk 12]
```diff
@@ -0,0 +1,404 @@
...
+		"type_code": "act"
+	},
+	{
+		"back_flavor": "You stand on a precipice before the Black Throne and gaze up at the blind, voiceless monstronsity that is Azathoth. The mere sight of it causes your vision to blur and your body to shake uncontrollably. The monotonous piping is so loud now, it rattles your bones and nearly causes you to vomit from the force upon your insides. Blood drips from your eyes and your ears.\nShould Azathoth awaken, it will devour every world that ever was. You must act now, before it is too late.",
+		"back_name": "The End",
+		"back_text": "Check Campaign Log.\n- If at least two of the following are true, you may proceed to <b>(→R2)</b>: <i>You accepted your fate</i>, <i>Mesmirizing Flute</i> is listed under \"Mementos Discovered,\" and <i>Ritual Components</i> is listed under \"Mementos Discovered.\"\n- If at least two of the following are true, you may proceed to <b>(→R3)</b>: <i>You rejected your fate</i>, <i>Scrap of Torn Shadow</i> is listed under \"Mementos Discovered,\" and <i>Wisp of Spectral Mist</i> is listed under \"Mementos Discovered.\"\n- Otherwise, <b>(→R4)</b>.",
+		"clues": 0,
```
Comment (drawntotheflame) [inline pack/tcu/bbt_encounter.json]: Please switch to null.

File: pack/tcu/bbt_encounter.json [hunk 13]
```diff
@@ -0,0 +1,404 @@
...
+		"encounter_code": "before_the_black_throne",
+		"faction_code": "mythos",
+		"flavor": "Your bones shiver violently with the silent, still cold of this empty place. No light penetrates these black halls. No warmth reaches you, no breeze touches your skin. This must be what death feels like, you think morbidly.",
+		"illustrator": "Reiko Murakami",
+		"name": "In Azathoth's Domain",
+		"pack_code": "bbt",
+		"position": 330,
+		"quantity": 1,
+		"stage": 2,
+		"text": "[action] Spend X clues: Draw the top X cards of the Cosmos. Choose one to put into play using its \"<b>Cosmos</b> -\" instructions, and move to it. Shuffle the rest back into the Cosmos.\n<b>Objective</b> - Only investigators at Court of the Great Old Ones may spend the requisite number of clues, as a group, to advance. (<i>Beware - any investigator who is not there will be killed.</i>",
```
Comment (drawntotheflame) [inline pack/tcu/bbt_encounter.json]: Please move the italics for Beware to before the (. Please add a closing ) after 'killed.'

File: pack/tcu/bbt_encounter.json [hunk 14]
```diff
@@ -0,0 +1,404 @@
...
+		"name": "The Final Countdown",
+		"pack_code": "bbt",
+		"position": 328,
+		"quantity": 1,
+		"stage": 3,
+		"text": "Each location is connected to each location adjacent to it.\nDo not remove doom from enemies when this agenda advances.",
+		"type_code": "agenda"
+	},
+	{
+		"back_flavor": "Your steed carries you through the entryway of the nightmarish palace, and you are surprised to find the halls inside to be as vast and cavernous as the empty space surrounding it. It is as though the doorway was but the threshold of another unvierse, as desolate and boundless as the last.",
```
Comment (drawntotheflame) [inline pack/tcu/bbt_encounter.json]: Typo: unvierse for universe.

File: pack/tcu/bbt_encounter.json [hunk 15]
```diff
@@ -0,0 +1,404 @@
...
+		"encounter_code": "before_the_black_throne",
+		"faction_code": "mythos",
+		"flavor": "From the horizonless breach in the gaping maw of the sky, you have traveled an impossible distance, and yet you are still many leagues from the nighted throne where Azathoth stirs.",
+		"illustrator": "Reiko Murakami",
+		"name": "The Cosmos Beckons",
+		"pack_code": "bbt",
+		"position": 329,
+		"quantity": 1,
+		"stage": 1,
+		"text": "[action] Spend X clues: Draw the top X cards of the Cosmos. Choose one to put into play using its \"<b>Cosmos</b> -\" instructions, and move to it. Shuffle the rest back into the Cosmos.\n<b>Objective</b> - Only investigators at Hideous Palace may spend the requisite number of clues, as a group, to advance. (<i>Beware - any investigator who is not there will be killed.</i>)",
```
Comment (drawntotheflame) [inline pack/tcu/bbt_encounter.json]: Please can you move the brackets outside the italics tags for the Beware clause?

File: pack/tcu/bbt_encounter.json [hunk 16]
```diff
@@ -0,0 +1,404 @@
...
+		"name": "It Awaits",
+		"pack_code": "bbt",
+		"position": 327,
+		"quantity": 1,
+		"stage": 2,
+		"text": "Each location is connected to each location adjacent to it.\nDo not remove doom from enemies when this agenda advances.",
+		"type_code": "agenda"
+	},
+	{
+		"back_name": "Azathoh Stirs",
```
Comment (drawntotheflame) [inline pack/tcu/bbt_encounter.json]: Typo: Azathoh for Azathoth

File: pack/tcu/bbt_encounter.json [hunk 17]
```diff
@@ -0,0 +1,404 @@
...
+		"back_text": "Each [[Cultist]] enemy commits ritual suicide. Discard each [[Cultist]] enemy in play and move all doom from them to Azathoth.\nThe lead investigator must search the encounter deck and discard pile for a copy of Daemonic Piping and draw it.\nThere is no longer an agenda deck, but the scenario is not over. Until the end of the scenario, doom that would be placed on the current agenda is instead placed on Azathoth, and all doom in play is considered to be on Azathoth.\n<hr><blockquote><i>You come across another circle of Azathoth's worshippers, led by a hooded, shapeless form which holds an ancient lantern high above its head. The lantern's dim grey light scarcely pierces the realm's darkness. Somehow, the phantom is familiar to you; some vague sense of kinship drives you onward.</i></blockquote>\nCheck Campaign Log. If <i>Corn Husk Doll</i> is listed under \"Mementos Discovered,\" the creature recognizes you, as well. It peers at you curiously, and the circle of prayer is silenced; remove 1 doom from Azathoth.\nOtherwise, the creature barely notices you, and the circle resumes its chanting.",
```
Comment (drawntotheflame) [inline pack/tcu/bbt_encounter.json]: Please remove the final \n in this line - it's surplus!

File: pack/tcu/bbt_encounter.json [hunk 18]
```diff
@@ -0,0 +1,404 @@
...
+		"encounter_code": "before_the_black_throne",
+		"faction_code": "mythos",
+		"flavor": "...that leering, tittering shantak coursed on impetuous and relentless, flapping its great slippery wings in malignant joy, and headed for those unhallowed pits whither no dreams reach... \n– H. P. Lovecraft, The Dream-Quest of Unknown Kadath",
+		"illustrator": "Lenka Šimečková",
+		"name": "The Final Countdown",
+		"pack_code": "bbt",
+		"position": 328,
+		"quantity": 1,
+		"stage": 3,
+		"text": "Each location is connected to each location adjacent to it.\nDo not remove doom from enemies when this agenda advances.",
```
Comment (drawntotheflame) [inline pack/tcu/bbt_encounter.json]: Please add in italics: '(Beware - the scenario will not end when this agenda advances, but the doom of the world will be close at hand.)'

Comment (drawntotheflame) [reply pack/tcu/bbt_encounter.json]: ... at the end of the line, I mean!

File: pack/tcu/bbt_encounter.json [hunk 19]
```diff
@@ -0,0 +1,404 @@
...
+		"pack_code": "bbt",
+		"position": 326,
+		"quantity": 1,
+		"stage": 1,
+		"text": "Each location is connected to each location adjacent to it.\nDo not remove doom from enemies when this agenda advances.",
+		"type_code": "agenda"
+	},
+	{
+		"back_name": "Gnawing Hungrily",
+		"back_text": "Each [[Cultist]] enemy commits ritual suicide. Discard each [[Cultist]] enemy in play and move all doom from them to Azathoth.\nThe lead investigator must search the encounter deck and discard pile for a copy of Daemonic Piping and draw it.\n<hr><blockquote><i>In this oblivion outside time, nothing is as it should be. The mist dances and churns around you. Laughter mocks you from beyond, and out from the dark nebulae emerges a familiar wraith, a being which should be long dead.</i></blockquote>\nCheck Campaign Log. If <i>Worn Crucifix</i> is listed under \"Mementos Discovered,\" the wraith recognizes the crucifix you stole from Walter Gilman's room and backs away slowly, slinking into the darkness&nbsp;beyond.\nOtherwise, the wraith attacks you; each investigator must test [willpower] (4). Each investigator who fails takes 1 damage and 1 horror.",
```
Comment (drawntotheflame) [inline pack/tcu/bbt_encounter.json]: Please remove the odd characters between 'darkness' and 'beyond'.

File: pack/tcu/bbt_encounter.json [hunk 20]
```diff
@@ -0,0 +1,404 @@
+[
+	{
+		"back_text": "Hard / Expert\n[skull]: -X. X is the amount of doom on Azathoth, to a minimum of 2.\n[cultist]: Reveal another token. If you fail, search the encounter deck and discard pile for a [[Cultist]] enemy and draw it. Shuffle the encounter deck.\n[tablet]: -3. If you fail, Azathoth attacks you.\n[elder_thing]: -6. If your modified skill value for this test is 0, place 1 doom on Azathoth.",
+		"code": "05025",
```
Comment (drawntotheflame) [inline pack/tcu/bbt_encounter.json]: Code incorrect. Correct is '05325'

File: pack/tcu/bbt_encounter.json [hunk 21]
```diff
@@ -0,0 +1,404 @@
...
+		"double_sided": true,
+		"encounter_code": "before_the_black_throne",
+		"faction_code": "mythos",
+		"name": "Before the Black Throne",
+		"pack_code": "bbt",
+		"position": 25,
```
Comment (drawntotheflame) [inline pack/tcu/bbt_encounter.json]: Position incorrect. Correct is '325'

File: pack/tcu/bbt_encounter.json [hunk 22]
```diff
@@ -0,0 +1,404 @@
...
+		"position": 332,
+		"quantity": 1,
+		"shroud": 2,
+		"text": "<b>Forced</b> - At the end of the round: Add clues to Cosmic Ingress until it has 3 clues on it.\nEach [[Void]] location gains: \"[action]: Shuffle this location into the Cosmos, moving each investigator and enemy that was at this location to Cosmic Ingress.\"",
+		"traits": "Otherworld.",
+		"type_code": "location"
+	},
+	{
+		"back_illustrator": "Uwe Jarling",
+		"back_name": "Comsos",
```
Comment (akaan) [inline pack/tcu/bbt_encounter.json]: Typo "Comsos" instead of "Cosmos". You may have the same typo elsewhere.

Comment (drawntotheflame) [reply pack/tcu/bbt_encounter.json]: Lol I think I completely blanked on that... :/

File: pack/tcu/bbt_encounter.json (hunk 1 above)
Comment (zzorba) [reply pack/tcu/bbt_encounter.json]: Done

**Pull request merged**